play/binary-explorer/index.html

```diff
@@ -63,12 +63,17 @@ <h1 class="text-center mt-2">
 				<button class="btn btn-primary" onclick="addRow()">Add Letter</button>
 				<button class="btn btn-success" onclick="addToMessage()">Add Letters to Message</button>
 				<button class="btn btn-warning" onclick="deleteMessage()">Delete Message</button>
+				<button class="btn btn-secondary" onclick="copyMessage()">Copy</button>
```

Comment:
Everything in this PR looks great, but could you perhaps change this button to be a little more descriptive? Perhaps "Copy Message" and include a relevant icon from [Font Awesome](https://fontawesome.com/icons?d=gallery&q=copy)?

Reply:
Will do